Nur in konsole/fonts: 9x15.pcf.gz.
Nur in konsole/fonts: console8x16.pcf.gz.
diff -urp konsole.bak/konsole/konsole.cpp konsole/konsole/konsole.cpp
--- konsole.bak/konsole/konsole.cpp	2006-11-05 15:47:21.000000000 +0100
+++ konsole/konsole/konsole.cpp	2006-11-05 16:53:51.000000000 +0100
@@ -132,6 +132,7 @@ Time to start a requirement list.
 #include <ktip.h>
 #include <kprocctrl.h>
 #include <ktabwidget.h>
+#include <qwidgetstack.h>
 #include <kregexpeditorinterface.h>
 #include <kparts/componentfactory.h>
 #include <kcharsets.h>
@@ -321,6 +322,9 @@ Konsole::Konsole(const char* name, int h
   makeTabWidget();
   setCentralWidget(tabwidget);
 
+  QBoxLayout* lay = (QBoxLayout*)this->layout();
+  lay->setSpacing(-1);
+
   if (b_dynamicTabHide || n_tabbar==TabNone)
     tabwidget->setTabBarHidden(true);
 
@@ -895,6 +899,9 @@ void Konsole::makeTabWidget()
   tabwidget->setAutomaticResizeTabs( b_autoResizeTabs );
   tabwidget->setTabCloseActivatePrevious( true );
 
+  QWidgetStack* stack = (QWidgetStack*)tabwidget->child("tab pages");
+  stack->setLineWidth(0);
+
   if (n_tabbar==TabTop)
     tabwidget->setTabPosition(QTabWidget::Top);
   else
Nur in konsole/konsole: konsole_dummy.cpp.
Nur in konsole/konsole: konsole.la.cpp.
diff -urp konsole.bak/konsole/main.cpp konsole/konsole/main.cpp
--- konsole.bak/konsole/main.cpp	2006-11-05 15:47:20.000000000 +0100
+++ konsole/konsole/main.cpp	2006-11-05 16:27:12.000000000 +0100
@@ -48,7 +48,7 @@
 
 // COMPOSITE disabled by default because the QApplication constructor
 // needed to enable the ARGB32 visual has undesired side effects.
-#if 0
+#if 1
 #if defined(Q_WS_X11) && defined(HAVE_XRENDER) && QT_VERSION >= 0x030300
 # define COMPOSITE
 #endif

